anago

```diff
@@ -203,25 +203,29 @@ check_prerequisites () {
        (logrun gcloud config set account $user && \
         logrun gcloud docker -- version >/dev/null 2>&1); then
       logecho -r "$OK"
+      break
```

Comment:
Yes, the original design automated releases for Googlers and we've been making incremental changes since to allow non-Googlers to run mock and true releases.  The details you point out for how to construct and use the full email are again just incremental moves toward that goal, but not fully functional.  Please do improve and help move us toward non-Googler run releases!